CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/server/RequestHandler/EncryptionKeyRequestHandler.java

```diff
@@ -0,0 +1,21 @@
+package com.couchbase.CouchbaseLiteServ.server.RequestHandler;
+
+
+import com.couchbase.CouchbaseLiteServ.server.Args;
+import com.couchbase.lite.EncryptionKey;
+
+public class EncryptionKeyRequestHandler {
```

Comment:
This is commented in ios

Reply:
feature is ready in Android, iOS will be ready soon